visualization/app/codeCharta/ui/ribbonBar/colorMetricChooser/colorMetricChooser.component.html

```diff
@@ -1,9 +1,10 @@
-<i class="fa fa-paint-brush"></i>
+<i [style.color]="(isHeightAndColorMetricLinked$ | async) ? disabledColor : nonDisabledColor" class="fa fa-paint-brush"></i>
```

Comment:
@shaman-apprentice Would do you think about my implementation? Is there a better way to change the icon color?

Reply:
I personally would not set those values in the component but inline the values here in the template. But this is fine as well.